### DISCUSSION TITLE
Add support for multiple docker-compose files

### DISCUSSION THREAD

**Discussion by jakob30061** [coollabsio/coolify#3188] [Feature Requests]
Hey,
I extend a base docker-compose depending on the environment with different containers and settings. It would be great adding a coma separated list of docker compose files (similar to the domain input field).
An info popup would be important as well.

Backend wise this command generates a merged docker compose file which could be parsed as normal.
`docker compose -f docker-compose-base.yml -f docker-compose-prod.yml config --no-interpolate --no-path-resolution`

The merged compose file generated by the command above has a slightly different format: Instead of 
```
ports:
    - 8055:8055 
``` 

the ports look like this:
```
ports:
    - mode: ingress
      protocol: tcp
      published: "8055"
      target: 8055 
```

same issue for volumes. I'm not sure if this could be an issue.
If this is wanted (I would love it) i could try to implement it. But never done Laravel before :)

Thanks to all maintainers of this create tool!

**Comment by Zyles** [comment]
Any success with this?

**Reply by djsisson** [reply]
you can run custom build/start commands for compose apps, so you can already pass in multiple files with -f

however if you mean from the point of view of having variable services, within coolify that you want coolify to alter, then yes this isn't currently possible
